contracts/controller/src/prices.rs

```diff
@@ -43,26 +54,38 @@ mod tests {
         return (eth_contract, token_address, user_account);
     }
 
-    use super::*;
-
     #[test]
     fn test_add_get_price() {
```

Comment:
@dmytro-shash please create a separate unit-test that checks that method produce correct output

Reply:
splitted into two diff tests as a outcome 
![image](https://user-images.githubusercontent.com/97089150/159912240-22cfe7b1-53a4-4886-8c69-cd9b7727ba90.png)